packages/mesos-dns/buildinfo.json

```diff
@@ -1,10 +1,10 @@
 {
-  "sources": {
-    "mesos-dns": {
-      "kind": "url",
-      "url": "https://github.com/mesosphere/mesos-dns/releases/download/v0.6.0/mesos-dns-v0.6.0-linux-amd64",
-      "sha1": "da98d671ca743ab44aa876b65d4e08db22b8c4d3"
-    }
+  "docker": "golang:1.7",
+  "single_source" : {
+    "kind": "git",
+    "git": "https://github.com/mesosphere/mesos-dns.git",
+    "ref": "919851a158ba71aee4de145dc0a7f90cc68d0902",
```

Comment:
This is 2 commits behind current master of mesos-dns, excluding "return errTruncated after setting header bits". Is that intentional?
I assume we're ok with using mesos-dns master instead of cherry-picking specific commits onto a 1.8 branch?